.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and run docker compose
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  docker:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    env:
+      CI_COMMIT_MESSAGE: Continuous Integration Build Artifacts
+      CI_COMMIT_AUTHOR: Continuous Integration
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Start containers
+      run: |
+        docker compose pull
+        make initialize_docker
```

Comment:
Can you avoid this and run the extraction on the host ? We run it inside a fully built local env because it's done manually by us, but it doesn't need a database, memcached, celery or even nginx, so that's just a lot of wasted CPU, especially with initialize creating base data as well.

Reply:
I will try it. I also wanted to see if I could run tests, which i can.

Reply:
```
service "web" is not running
```

Reply:
Will try starting without initializing.

Reply:
mmm the script currently assumes it's running on a host where docker containers run, so it would need some tweaking, yeah. Avoiding initializing is a good step, but ideally we wouldn't even have docker at all here.

Reply:
I think there is a lot of room for improvement there, but if it takes 10 minutes but runs successfully and doesn't require any kind of auth support and also doesn't have security holes for forks, then I would bet on this. The upside is there and it has none of the downsides AFAICT.